source/agora/test/Base.d

```diff
@@ -2312,7 +2313,18 @@ public APIManager makeTestNetwork (APIManager : TestAPIManager = TestAPIManager)
       network : net.nodes.map!(pair => "http://"~pair.address).array.idup,
       logging: test_conf.logging,
       event_handlers : test_conf.event_handlers,
-      registry: { enabled: true, },
+      registry: {
+          validators: {
+              authoritative: true,
+              primary: "test.name.registry",
+              soa: { email: SetInfo!string("test@testnet"), },
+          },
+          flash: {
+              authoritative: true,
+              primary: "test.name.registry",
```

Comment:
Is that correct ? Shouldn't that be `name.registry`, as this one is the authoritative registry ?